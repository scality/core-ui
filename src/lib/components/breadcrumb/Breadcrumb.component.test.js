import initStoryshots from "@storybook/addon-storyshots";

initStoryshots({
  configPath: "src/lib/components/breadcrumb/__snapshots__"
});
