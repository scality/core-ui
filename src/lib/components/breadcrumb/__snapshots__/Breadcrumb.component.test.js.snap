// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Breadcrumb Default 1`] = `
<div>
  <h3>
    Default Breadcrumb 
  </h3>
  <ol
    className="sc-breadcrumb sc-bdVaJa uRaVh"
  >
    <li
      className="sc-breadcrumb_item sc-bwzfXH ivdgiW"
    >
      <a
        href="home"
      >
        home
      </a>
    </li>
    <li
      className="sc-breadcrumb_separator sc-htpNat fmvEAc"
    >
      <i
        className="fas fa-chevron-right"
      />
    </li>
    <li
      className="sc-breadcrumb_item sc-bwzfXH ivdgiW"
    >
      <a
        href="clusters"
      >
        clusters
      </a>
    </li>
    <li
      className="sc-breadcrumb_separator sc-htpNat fmvEAc"
    >
      <i
        className="fas fa-chevron-right"
      />
    </li>
    <li
      className="sc-breadcrumb_item sc-bwzfXH ivdgiW"
    >
      <a
        href="cluster"
      >
        cluster_1
      </a>
    </li>
    <li
      className="sc-breadcrumb_separator sc-htpNat fmvEAc"
    >
      <i
        className="fas fa-chevron-right"
      />
    </li>
    <li
      className="sc-breadcrumb_item sc-bwzfXH ivdgiW"
    >
      <a
        href="nodes"
      >
        nodes
      </a>
    </li>
    <li
      className="sc-breadcrumb_separator sc-htpNat fmvEAc"
    >
      <i
        className="fas fa-chevron-right"
      />
    </li>
    <li
      className="sc-breadcrumb_item sc-bwzfXH ivdgiW"
    >
      <a
        href="node"
        title="node_longlonglonglonglonglonglonglonglonglonglong"
      >
        node_longlonglonglonglonglonglonglonglonglonglong
      </a>
    </li>
    <li
      className="sc-breadcrumb_separator sc-htpNat fmvEAc"
    >
      <i
        className="fas fa-chevron-right"
      />
    </li>
    <li
      className="sc-breadcrumb_item sc-bwzfXH ivdgiW"
    >
      <a
        href="volumes"
      >
        volumes
      </a>
    </li>
    <li
      className="sc-breadcrumb_separator sc-htpNat fmvEAc"
    >
      <i
        className="fas fa-chevron-right"
      />
    </li>
    <li
      className="sc-breadcrumb_item sc-bwzfXH eZtnXH"
    >
      <label>
        volume_1
      </label>
    </li>
  </ol>
  <h3>
    Breadcrumb with customized color
  </h3>
  <ol
    className="sc-breadcrumb sc-bdVaJa uRaVh"
  >
    <li
      className="sc-breadcrumb_item sc-bwzfXH cgzeoU"
    >
      <a
        href="home"
      >
        home
      </a>
    </li>
    <li
      className="sc-breadcrumb_separator sc-htpNat fmvEAc"
    >
      <i
        className="fas fa-chevron-right"
      />
    </li>
    <li
      className="sc-breadcrumb_item sc-bwzfXH cgzeoU"
    >
      <a
        href="clusters"
      >
        clusters
      </a>
    </li>
    <li
      className="sc-breadcrumb_separator sc-htpNat fmvEAc"
    >
      <i
        className="fas fa-chevron-right"
      />
    </li>
    <li
      className="sc-breadcrumb_item sc-bwzfXH cgzeoU"
    >
      <a
        href="cluster"
      >
        cluster_1
      </a>
    </li>
    <li
      className="sc-breadcrumb_separator sc-htpNat fmvEAc"
    >
      <i
        className="fas fa-chevron-right"
      />
    </li>
    <li
      className="sc-breadcrumb_item sc-bwzfXH cgzeoU"
    >
      <a
        href="nodes"
      >
        nodes
      </a>
    </li>
    <li
      className="sc-breadcrumb_separator sc-htpNat fmvEAc"
    >
      <i
        className="fas fa-chevron-right"
      />
    </li>
    <li
      className="sc-breadcrumb_item sc-bwzfXH cgzeoU"
    >
      <a
        href="node"
        title="node_longlonglonglonglonglonglonglonglonglonglong"
      >
        node_longlonglonglonglonglonglonglonglonglonglong
      </a>
    </li>
    <li
      className="sc-breadcrumb_separator sc-htpNat fmvEAc"
    >
      <i
        className="fas fa-chevron-right"
      />
    </li>
    <li
      className="sc-breadcrumb_item sc-bwzfXH cgzeoU"
    >
      <a
        href="volumes"
      >
        volumes
      </a>
    </li>
    <li
      className="sc-breadcrumb_separator sc-htpNat fmvEAc"
    >
      <i
        className="fas fa-chevron-right"
      />
    </li>
    <li
      className="sc-breadcrumb_item sc-bwzfXH ipfFVV"
    >
      <label>
        volume_1
      </label>
    </li>
  </ol>
</div>
`;
